kwave/__init__.py

```diff
@@ -15,6 +15,7 @@
 # is refactored
 
 platform = sys.platform
+platform = 'linux'
```

Comment:
This was for local debugging no? and can be removed from the PR?

Reply:
Good catch! Removed it